canonical/ubuntu.com
# Cred PoC syllabus updates

## Done

- Prettify syllabus pages
- Fix the base template for cube pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- visit [https://ubuntu-com-12023.demos.haus/credentialing/syllabus](https://ubuntu-com-12023.demos.haus/credentialing/syllabus)
- Shouldnot look like dumpster fire

## Issue / Card

Fixes #

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
